app.js

```diff
@@ -30,41 +31,43 @@ var people = askQuestion('Do I like people? (answer yes or no)', 'You are correc
 function numberQuestion() {
   var number = 7;
   var notDone = true;
-  var rightNum;
-  for (var num = 4; num > 0 && notDone; num--){
-    var rightNum = prompt('What is my favorite number?');
-    rightNum = parseInt(rightNum);
+  var guessNum;
+  for (var i = 4; i > 0 && notDone; i--){
+    guessNum = prompt('What is my favorite number?');
+    guessNum = parseInt(guessNum);
 
-    if(rightNum === number){
+    if(guessNum === number){
       alert('Wow! You guessed the correct number!!');
       notDone = false;
+      console.log('number answered correctly');
 
-    }else if(num === 1){
+    }else if(i === 1){
       alert('You ran out of guesses!');
 
-    }else if(rightNum > number){
+    }else if(guessNum > number){
       alert('You\'re too high!');
 
     }else{
       alert('You\'re too low!');
     }
   }
-  return rightNum;
+  return guessNum;
 }
-var rightNum = numberQuestion();
+var guessNum = numberQuestion();
```

Comment:
The variable above is inside the function. I renamed the variable outside the function to clear up confusion